apps/wallet/src/ui/app/components/menu/content/MenuList.tsx

```diff
@@ -45,16 +48,34 @@ function MenuList() {
 	const navigate = useNavigate();
 	const [logoutInProgress, setLogoutInProgress] = useState(false);
 	const [isLogoutDialogOpen, setIsLogoutDialogOpen] = useState(false);
+	const explorerAddress = useExplorerLink({
+		type: ExplorerLinkType.address,
+		address: address || '',
+	});
 	return (
 		<>
 			<MenuLayout title="Wallet Settings">
 				<div className="flex flex-col divide-y divide-x-0 divide-solid divide-gray-45">
-					<MenuListItem
-						to={accountUrl}
-						icon={<Account24 />}
-						title={'Accounts'}
-						subtitle={domainName ?? (address ? formatAddress(address) : '')}
-					/>
+					<div className="flex flex-col mb-3">
+						<MenuListItem
+							to={accountUrl}
+							icon={<Account24 />}
+							title={'Accounts'}
+							subtitle={domainName ?? (address ? formatAddress(address) : '')}
+						/>
+
+						<Button
+							variant="secondary"
+							size="narrow"
+							href={explorerAddress || ''}
```

Comment:
Should we event render this if somehow the `explorerAddress` is empty?

Reply:
Yes, it should never be the case, since a use must be logged in to access wellet settings